Makefile

```diff
@@ -262,13 +262,19 @@ else
 	ARKOUDA_COMPAT_MODULES += -M $(ARKOUDA_SOURCE_DIR)/compat/lt-125
 endif
 
+ifdef ARKOUDA_SERVER_USER_MODULES
+	ifeq (,$(findstring \-M,$(ARKOUDA_SERVER_USER_MODULES)))
```

Comment:
does this grock a list of directories contained in the `ARKOUDA_SERVER_USER_MODULES` env var?

Reply:
so we can include modules from multiple directories.